model/sparql/GenericSparql.php

```diff
@@ -2243,22 +2242,31 @@ public function listConceptGroupContents($groupClass, $group, $lang,$showDepreca
      * @param string $lang language of labels to return
      * @param int $offset offset of results to retrieve; 0 for beginning of list
      * @param int $limit maximum number of results to return
+     * @param boolean $showDeprecated whether to include deprecated concepts in the change list
      * @return string sparql query
      */
-    private function generateChangeListQuery($prop, $lang, $offset, $limit=200) {
+    private function generateChangeListQuery($prop, $lang, $offset, $limit=200, $showDeprecated=false) {
         $fcl = $this->generateFromClause();
         $offset = ($offset) ? 'OFFSET ' . $offset : '';
 
+        //Additional clauses when deprecated concepts need to be included in the results
+        $deprecatedVars = ($showDeprecated) ? '?replacedBy ?deprecated ?replacingLabel' : '';
+        $deprecatedOptions = ($showDeprecated) ? 'UNION { ?concept dc:isReplacedBy ?replacedBy ; dc:modified ?date2 . BIND(COALESCE(?date2, ?date) AS ?date) OPTIONAL { ?replacedBy skos:prefLabel ?replacingLabel . FILTER (langMatches(lang(?replacingLabel), \''.$lang.'\')) }} OPTIONAL { ?concept owl:deprecated ?deprecated . }' : '';
```

Comment:
This is (by far) the longest line in the file. Worth considering: split? (increases readability)